src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1SpaceBookingService.kt

```diff
@@ -438,6 +438,10 @@ class Cas1SpaceBookingService(
       else -> throw InternalServerErrorProblem("Withdrawal triggered automatically is not supported")
     }
     cas1BookingDomainEventService.spaceBookingCancelled(spaceBooking, user, reason)
+    cas1ApplicationStatusService.spaceBookingCancelled(
+      spaceBooking,
+      isWithdrawableTreeOperation = withdrawalContext.triggeringEntityType == WithdrawableEntityType.PlacementRequest,
```

Comment:
I don't think it's the correct answer, but doing something similar to BookingService would at least maintain the existing behaviour? e.g.

```
isUserRequestedWithdrawal = withdrawalContext.triggeringEntityType == WithdrawableEntityType.SpaceBooking,
```

And then ignore the change if isUserRequestedWithdrawal?